Selenium/selenium6.rb

```diff
@@ -0,0 +1,18 @@
+require 'selenium-webdriver'
+
+# Open Chrome Browser
+
+Selenium::WebDriver::Chrome::Service.driver_path = 'C:\Users\neera.yadav\Documents\BrowserDrivers\chromedriver-win64\chromedriver-win64\chromedriver.exe'
+driver = Selenium::WebDriver.for :chrome
+
+driver.get 'https://letcode.in/forms'
+
+dropdown = driver.find_element(:xpath , '/html/body/app-root/app-forms/section[1]/div/div/div[1]/div/div/form/div[5]/div[2]/div/div/div/select')
+
+options = dropdown.find_elements(tag_name: 'option')
+
```

Comment:
A better approach is to use

`dropdown = Selenium::WebDriver::Support::Select.new(dropdown_xpath)`

You should make use of the functions mentioned here: https://www.selenium.dev/selenium/docs/api/rb/Selenium/WebDriver/Support/Select.html

Reply:
Made the changes

Reply:
Done